Core/src/main/java/com/paypal/android/core/API.kt

```diff
@@ -14,4 +14,17 @@ class API internal constructor(
             httpRequestFactory.createHttpRequestFromAPIRequest(apiRequest, configuration)
         return http.send(httpRequest)
     }
+
+    suspend fun getClientId(): String {
+        val apiRequest = APIRequest(
+            path = "v1/oauth2/token",
+            method = HttpMethod.GET,
+            body = null
```

Comment:
nit: `APIRequest` can have `body=null` since get requests will not have

Reply:
👍 that's a good call I'll update.